### ISSUE TITLE
Assign Users

### ISSUE THREAD

**Issue by sladyn98** [MichaelMure/git-bug#112]
With the identity section almost done woudn't it make sense to add the ability to assign bugs to a user .
`assign bug <id> <username>`
Some of the things we need to take care of:
1) Only the author of the bug is allowed to assign it to someone.
2) Section of assignees when the snapshot of the bug is compiled.
3) Addition to the GraphQL API.

**Comment by MichaelMure** [comment]
Yes that's a feature supported by pretty much all bug tracker (cf https://github.com/MichaelMure/git-bug/issues/30).

Let's do it as well.
